src/large-int.js

```diff
@@ -0,0 +1,119 @@
+import { XdrPrimitiveType } from './xdr-type';
+import { calculateBigIntBoundaries, encodeBigIntFromBits, sliceBigInt } from './bigint-encoder';
+import { XdrNotImplementedDefinitionError, XdrWriterError } from './errors';
+
+export class LargeInt extends XdrPrimitiveType {
+  /**
+   * @param {Array<Number|BigInt|String>} parts - Slices to encode
+   */
+  constructor(args) {
+    super();
```

Comment:
to enforce abstract at class level, should we add the check:
```
if (this.constructor === LargeInt) {
    throw new Error("cannot create abstract class")
}
```

Reply:
That's the performance-critical code, so my intention was to reduce the amount of (probably) excessive typechecks in `read`/`write` functions. Have no strong objections regarding the constructor.

Reply:
Had a similar thought re: read/write